projects/components/src/popover/popover.component.ts

```diff
@@ -76,17 +77,17 @@ export class PopoverComponent implements OnDestroy {
     });
 
     // Closing can happen internal to the Popover for things like closeOnBackdropClick. Let the consumer know.
-    this.subscription = popover.closed$.subscribe(() => this.popoverClose.emit());
+    this.popover.closed$.subscribe(() => this.popoverClose.emit());
```

Comment:
Why are not unsubscribing from this?

Reply:
No longer necessary, because the closed observable ends when we invoke close which now will always hope at or before destroy.